src/async_action.cpp

```diff
@@ -45,4 +45,13 @@ void time_handler(BPFtrace *bpftrace, void *data)
   bpftrace->out_->message(MessageType::time, timestr, false);
 }
 
+void helpererror_handler(BPFtrace *bpftrace, void *data)
```

Comment:
nit
```suggestion
void helper_error_handler(BPFtrace *bpftrace, void *data)
```
As you can probably tell I'm a stickler for readability 👓

Reply:
Done. 😄